### PR TITLE
Refactor scheduler_perf_test codes

### PR DESCRIPTION
**Pull request by kerthcet**
#### What type of PR is this?
/kind cleanup
/sig scheduling

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

Three commits here:
1) Add a new definition type for `operationCode`
2) Most about the annotations, make it more readability. Another change is replace if-else with switch-case clause.
3) Rename `SchedulerConfigFile` to `SchedulerConfigFilePath` and make it a pointer to be consistent with `DefaultPodTemplatePath`.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by kerthcet** [comment]
cc @alculquicondor

**Comment by alculquicondor** [comment]
Can you add some details on what this refactor is about in the description?

Also, maybe this can wait until after code freeze and before test freeze.

File: test/integration/scheduler_perf/scheduler_perf_test.go [hunk 1]
```diff
@@ -52,23 +52,30 @@ import (
...
+	createNodesOpcode      operationCode = "createNodes"
+	createNamespacesOpcode operationCode = "createNamespaces"
+	createPodsOpcode       operationCode = "createPods"
+	createPodSetsOpcode    operationCode = "createPodSets"
+	churnOpcode            operationCode = "churn"
+	barrierOpcode          operationCode = "barrier"
+	sleepOpcode            operationCode = "sleep"
+)
 
+const (
```
Comment (alculquicondor) [inline test/integration/scheduler_perf/scheduler_perf_test.go:67]: just leaving empty lines within the `const` block should be enough, but up to you.

File: test/integration/scheduler_perf/scheduler_perf_test.go [hunk 2]
```diff
@@ -84,27 +91,31 @@ var (
...
 	// createNodesOp, createPodsOp, or barrierOp.
 	WorkloadTemplate []op
 	// List of workloads to run under this testCase.
 	Workloads []*workload
-	// SchedulerConfigFile is the path of scheduler configuration
-	SchedulerConfigFile string
-	// Default path to spec file describing the pods to create. Optional.
+	// SchedulerConfigFilePath is the path of scheduler configuration
+	// Optional
+	SchedulerConfigFilePath *string
```
Comment (alculquicondor) [inline test/integration/scheduler_perf/scheduler_perf_test.go:115]:
```suggestion
	SchedulerConfigPath *string
```
?

**Review by alculquicondor — COMMENTED** [review]
/approve

File: test/integration/scheduler_perf/scheduler_perf_test.go (hunk 2 above)
Comment (kerthcet) [reply test/integration/scheduler_perf/scheduler_perf_test.go]: Updated and squashed.

File: test/integration/scheduler_perf/scheduler_perf_test.go (hunk 1 above)
Comment (kerthcet) [reply test/integration/scheduler_perf/scheduler_perf_test.go:67]: Block can help to declare a group of related constants or variables, prefer to keep it.

File: test/integration/scheduler_perf/scheduler_perf_test.go [hunk 3]
```diff
@@ -84,27 +91,31 @@ var (
...
 	// createNodesOp, createPodsOp, or barrierOp.
 	WorkloadTemplate []op
 	// List of workloads to run under this testCase.
 	Workloads []*workload
-	// SchedulerConfigFile is the path of scheduler configuration
-	SchedulerConfigFile string
-	// Default path to spec file describing the pods to create. Optional.
+	// SchedulerConfigPath is the path of scheduler configuration
+	// Optional
+	SchedulerConfigPath *string
```
Comment (alculquicondor) [inline test/integration/scheduler_perf/scheduler_perf_test.go:115]: Were there no YAML files using this field?

Comment (kerthcet) [reply test/integration/scheduler_perf/scheduler_perf_test.go:115]: Yes, search the `schedulerConfigFile` with no result return.

**Pull request merged**